### ISSUE TITLE
Tor alternative sources cannot be set in Debian 11.4

### ISSUE THREAD

**Issue by ktecho** [raspiblitz/raspiblitz#3229]
Tor alternative sources cannot be set in Debian 11.4. Then the rest of the install process fails because an APT source is malformed.

```
*** Adding Tor Sources ***

deb [arch=amd64] tor+http://apow7mjfryruh65chtdydfmqfpj5btws7nbocgtaovhvezgccyjazpqd.onion/torproject.org  main
deb-src [arch=amd64] tor+http://apow7mjfryruh65chtdydfmqfpj5btws7nbocgtaovhvezgccyjazpqd.onion/torproject.org   main

- OK sources added

*** Reinstall ***
E: Malformed entry 2 in list file /etc/apt/sources.list.d/tor.list (Component)
E: The list of sources could not be read.
E: Malformed entry 2 in list file /etc/apt/sources.list.d/tor.list (Component)
E: The list of sources could not be read.
E: Malformed entry 2 in list file /etc/apt/sources.list.d/tor.list (Component)
E: The list of sources could not be read.
E: Malformed entry 2 in list file /etc/apt/sources.list.d/tor.list (Component)
E: The list of sources could not be read.
E: Malformed entry 2 in list file /etc/apt/sources.list.d/tor.list (Component)
E: The list of sources could not be read.
```

The package `lsb-release` is not installed by default in the debian image that I'm using. It comes as the default template for installing Debian in Proxmox. So doing this:

```
apt install lsb-release
```

makes it so the tor sources are well formed. Should I add the `lsb-release` package to the list of packages to install in `build_sdcard.sh` and send a PR?

**Comment by openoms** [comment]
yes, that's a good solution and might help installing on other distros also

**Comment by rootzoll** [comment]
merged #3233 to fix - closing for v1.8.0 release
